src/main/java/io/github/fripe070/ghosthunting/block/LightswitchBlockEntity.java

```diff
@@ -8,6 +8,8 @@
 import net.minecraft.util.math.BlockPos;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.Arrays;
```

Comment:
Why was an import added if the only other change in the file was deleting two lines?

Reply:
Was a debuting step. Will fix tonight when I optimise imports